### DISCUSSION TITLE
Return of the dreaded 'noneType' for batch load image

### DISCUSSION THREAD

**Discussion by bobpuffer1** [WASasquatch/was-node-suite-comfyui#182] [Q&A]
I must admit I'm flummoxed in regards to using batch load image. 
I've turned on batches in the comfy UI controller 
My folder has got JPEG images in it and I'm searching for the wild card for JPG images 
I'm using an exact path 
But I continued to get The Nonetype object is not subscribable. 
I saw where this was an issue #81 And it was apparently patched But I'm getting it again

**Comment by bobpuffer1** [comment]
Error occurred when executing PreviewImage:

'NoneType' object is not subscriptable

File "C:\Users\Bob Puffer\ComfyUI_windows_portable\ComfyUI\execution.py", line 151, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
File "C:\Users\Bob Puffer\ComfyUI_windows_portable\ComfyUI\execution.py", line 81, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
File "C:\Users\Bob Puffer\ComfyUI_windows_portable\ComfyUI\execution.py", line 74, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
File "C:\Users\Bob Puffer\ComfyUI_windows_portable\ComfyUI\nodes.py", line 1265, in save_images
full_output_folder, filename, counter, subfolder, filename_prefix = folder_paths.get_save_image_path(filename_prefix, self.output_dir, images[0].shape[1], images[0].shape[0])

**Reply by WASasquatch** [reply]
This is something I still can't reproduce unless I have edited the files being searched. It index the files. So if you remove, rename, move images it will try to load it from path

**Reply by bobpuffer1** [reply]
I got it to work. I just checked everything and made sure my absolute path was absolutely correct now it's ready to go and I'll keep you posted with any other issues I run into

**Reply by bobpuffer1** [reply]
there is the known stupid issue with comfy UI where it pretty much ignores whatever you've got in that batch information you can set your batch to 8 or 12 or whatever and it never counts down and only one image gets processed from your batch. your workaround to just load up the queue works the best and then it cycles to the folder contents so that works retty good
